src/main/java/uk/gov/justice/hmpps/prison/service/AgencyService.java

```diff
@@ -223,22 +223,14 @@ public void verifyAgencyAccess(final String agencyId, boolean accessDeniedError,
         if (agencyIds.isEmpty()) {
```

Comment:
I will do a separate PR to tidy up this method - just realised I missed it (This will be similar to the change for bookingservice)